charts/kube-prometheus-stack.yaml

```diff
@@ -246,6 +246,9 @@ kube-state-metrics:
     registry: '__var__(repo.registry_endpoint)'
     repository: '__image_no_reg__(kube-state-metrics)'
 
+  extraArgs:
+    - --metric-labels-allowlist=persistentvolumeclaims=[excluded-from-alerts]
```

Comment:
There is a `metricLabelsAllowlist` field from what I see so I suggest to use this one instead